variables.tf

```diff
@@ -275,4 +275,31 @@ variable "logs_routing_tenant_regions" {
   nullable    = false
 }
 
+########################################################################
+# Cloud Logs Policies - TCO Optimizer
+#########################################################################
+
+variable "logs_policies_config" {
```

Comment:
suggest to call this `cloud_logs_policies`. All the other cloud logs variables start with `cloud_logs_` so lets be consistent (note that logs_routing is a different service)